app/screens/post_options/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PostOptions from './post_options';
+
+export default PostOptions;
```

Comment:
Is this in preparation for the wiring, or are we adding this index for some other reason?

Reply:
Preparation for wiring.